src/OpenTelemetry.Instrumentation.AspNetCore/CHANGELOG.md

```diff
@@ -35,6 +35,9 @@ exception. The attribute value will be set to full name of exception type.
 * Fixed `network.protocol.version` attribute values to match the specification.
   ([#5007](https://github.com/open-telemetry/opentelemetry-dotnet/pull/5007))
 
+* Removed filtering of metrics generated as a result of pull from Prometheus.
```

Comment:
This is one of the effects of the change, not the actual change!

We removed generating metrics if request path contained "metrics" - this is the change.

Reply:
updated